code/datums/ship.dm

```diff
@@ -343,8 +343,11 @@ var/next_ship_id
 
 	if(ship.target_system && ship.system && !ship.is_jumping)
 //		message_admins("[ship.name] [ship.system] [ship.mission_ai.cname] [ship.target_system] [ship.system]")
-		if(!ship.star_path.len) //dunno how this happens yet but it does
+		if(!ship.star_path || !ship.star_path.len) //dunno how this happens yet but it does
 			ship.star_path = get_path_to_system(ship.system, ship.target_system, ship.ftl_range, 200)
+			if(!ship.star_path.len) //If we still can't make path...
```

Comment:
but get_path_to_system will return at least clear list()
And runtimes here is because ship.star_path isn't initialized (or i can't find where it occurs).